vnext/ReactUWP/Utils/PropertyUtils.h

```diff
@@ -136,10 +141,14 @@ void UpdateCornerRadiusOnElement(ShadowNodeBase *node, const T &element) {
 template <class T>
 bool TryUpdateForeground(const T &element, const std::string &propertyName, const folly::dynamic &propertyValue) {
   if (propertyName == "color") {
-    if (IsValidColorValue(propertyValue))
-      element.Foreground(BrushFrom(propertyValue));
-    else if (propertyValue.isNull())
+    if (IsValidColorValue(propertyValue)) {
+      const auto brush = BrushFrom(propertyValue);
+      element.Foreground(brush);
+      UpdateTextControlForegroundResourceBrushes(element, brush);
```

Comment:
>UpdateTextControlForegroundResourceBrushes [](start = 6, length = 42)

Since we are in need to do the same thing for TextInput, Switch and CheckBox, can we call a generic UpdateControlForegroundResurceBrushes and then have specific element type functions inside it?

Reply:
Yeah, I think that's a good idea. Would you mind if I waited until the 2nd PR (Switch or CheckBox, whichever comes first) to introduce that layer of indirection?

Reply:
Yeah, sounds good to me.. I was just going to ask you if you could also fix the switch too... 🙂

Reply:
Actually, went ahead and incorporated now (commit 7). Thanks for the suggestion -- PropertyUtils.h looks better.

Will work on Switch/ToggleSwitch tomorrow.